README.md

```diff
@@ -19,10 +19,68 @@ Now, use the tool with `kubectl`
 kubectl rook-ceph ceph status
 ```
 
+## Usage
 
-## Commands
+`kubectl rook-ceph <root-args> <command> <command-args>`
 
-TBD
+### Root args
+
+- `--namespace` | `-n`: the Kubernetes namespace in which the CephCluster resides (default: rook-ceph)
+- `--help` | `-h`: Output help text
+
+### Commands
+
+- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
+
+- `operator`: run sub-commands against the Rook-Ceph operator
+  -  `restart`: restart the Rook-Ceph operator
+
+- `help`: Output help text
+
+
+## Examples
+
+### Run a Ceph Command
+
+Any `ceph` command can be run with the plugin. This example gets the ceph status
+
+```console
+kubectl rook-ceph ceph status
+```
+
+>```text
+>  cluster:
+>    id:     a1ac6554-4cc8-4c3b-a8a3-f17f5ec6f529
+>    health: HEALTH_WARN
```

Comment:
How about a healthy example? :) 
```suggestion
>    health: HEALTH_OK
```
And you can remove the following two warning lines